docs/solutions/logic-errors/tool-output-header-suppression-2026-05-02.md

```
@@ -0,0 +1,171 @@
+  - enum-driven
+  - header-suppression
+plan_ref: "issue-396-tool-output-appearance"
+---
+
+## Problem
+
+Tool call output displayed redundant headers (TUI: `→ tool_name`, CLI: `[tool] name`) even when `call_template` markdown already provided a concise description. This cluttered the transcript and reduced information density.
+
+## Symptoms
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)